spec/controllers/api/openid_connect/clients_controller_spec.rb

```diff
@@ -0,0 +1,134 @@
+require "spec_helper"
+
+describe Api::OpenidConnect::ClientsController, type: :controller do
+  describe "#create" do
+    context "when valid parameters are passed" do
+      it "should return a client id" do
+        stub_request(:get, "http://example.com/uris")
+          .with(headers: {Accept: "*/*", :"Accept-Encoding" => "gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
+                          :"User-Agent" => "Faraday v0.9.1"})
```

Comment:
Have you run RSpec to see if it fails? Looks like Travis didn't like it at all :/